Pipfile

```diff
@@ -9,9 +9,9 @@ python-jenkins = "*"
 
 [dev-packages]
 flake8 = "*"
-flake8-commas = "*"
 pydocstyle = "*"
 pylint = "*"
+black = "==19.3b0"
```

Comment:
Please keep this list sorted

Reply:
Also, how fragile is black anyways? Could we not just set this to `= “*”` and update it the same way as other packages? Or does the behavior of this software change significantly between versions?

Reply:
Addressed in c8c7b650683498652748482fc27779c70fc8cbe2.

Reply:
The setting it to `= "*"` won't work because `pipenv` hates "beta" software, and `black` is being extreeeeemly cautious about waiting to release a non-beta version.

Reply:
Thanks for the clarification btw!